src/pages/workspaces/workspace/DeleteWorkspaceModal.js

```diff
@@ -92,14 +98,23 @@ const DeleteWorkspaceModal = ({ workspace: { workspace: { namespace, name, bucke
         ...Utils.newTabLinkProps,
         href: bucketBrowserUrl(bucketName)
       }, ['Google Cloud Bucket']),
-      ' and all its data.'
+      ' and all its data',
+      workspaceBucketUsageInBytes !== undefined && span({ style: { fontWeight: 600 } }, ` (${Utils.formatBytes(workspaceBucketUsageInBytes)})`),
+      '.'
     ]),
     hasApps() && div({ style: { marginTop: '1rem' } }, [
       p(['Deleting it will also delete any associated applications:']),
       getAppDeletionMessage()
     ]),
+    collaboratorEmails && collaboratorEmails.length > 0 && div({ style: { marginTop: '1rem' } }, [
+      p(`${pluralize('collaborator', collaboratorEmails.length, true)} will lose access to this workspace.`),
+      div(collaboratorEmails.map(
+        email => div({ key: email, style: { overflow: 'hidden', textOverflow: 'ellipsis' } }, [h(Link, { href: `mailto:${email}` }, [email])])
```

Comment:
What happens if there is a very long list of collaborators?

Reply:
Done in https://github.com/DataBiosphere/terra-ui/pull/2843/commits/9b17fc95d44c3f984a4dc9516a508765ebd78427.

Reply:
Thanks for making this change! You have to checkmarks, so you are good to go.